src/block/inventory/FurnaceInventory.php

```diff
@@ -84,13 +84,19 @@ public function canAddSmelting(Item $item) : bool{
 	}
 
 	public function canAddFuel(Item $item) : bool{
-		if($item->getFuelTime() == 0) return false;
+		if($item->getFuelTime() == 0){
```

Comment:
```suggestion
		if($item->getFuelTime() === 0){
```
`==` should almost always be avoided.